tps-client/src/rpc_client.rs

```diff
@@ -18,7 +19,15 @@ use {
 
 impl TpsClient for RpcClient {
     fn send_transaction(&self, transaction: Transaction) -> TpsClientResult<Signature> {
-        RpcClient::send_transaction(self, &transaction).map_err(|err| err.into())
+        RpcClient::send_transaction_with_config(
+            self,
+            &transaction,
+            RpcSendTransactionConfig {
+                skip_preflight: true,
+                ..Default::default()
+            },
```

Comment:
does it make sense to make this configurable via CLI or something?

Reply:
Well, if you believe it is valuable, I can add this. But with current implementation it is not possible because TpsClient trait is implemented for RpcClient structure. So adding the flag would require using some wrapper for RpcClient which holds both RpcClient and this flag.

Reply:
ahh ya right this is just for bench tps. so `skip_preflight` makes sense. got this confused with the actual, not bench-tps rpc client